web/src/components/organisms/Settings/Workspace/index.tsx

```diff
@@ -0,0 +1,198 @@
+import styled from "@emotion/styled";
+import Button from "@reearth-cms/components/atoms/Button";
+import Form from "@reearth-cms/components/atoms/Form";
+import Input from "@reearth-cms/components/atoms/Input";
+import List from "@reearth-cms/components/atoms/List";
+import React, { useCallback, useEffect, useState } from "react";
+import { useNavigate, useParams } from "react-router-dom";
+
+import useHooks, { RoleUnion } from "./hooks";
+
+const Workspace: React.FC = () => {
+  const { workspaceId } = useParams();
+  const {
+    me,
+    currentWorkspace,
+    searchedUser,
+    changeSearchedUser,
+    deleteWorkspace,
+    searchUser,
+    updateName,
+    addMembersToWorkspace,
+    updateMemberOfWorkspace,
+    removeMemberFromWorkspace,
+  } = useHooks({ workspaceId });
+  const [owner, setOwner] = useState(false);
+  const [memberName, setMemberName] = useState<string>("");
+  const [workspaceName, setWorkspaceName] = useState<string>("");
+  const navigate = useNavigate();
+  const members = currentWorkspace?.members;
```

Comment:
The top of components and hooks often gets messy. Can you arrange these and space them out a little more cleanly? (I typically group third party hooks, useHooks, and stateful things separately. But its up to you since it depends on the comp/hooks file)